app-android/src/main/java/com/ustadmobile/port/android/view/HolidayCalendarEditFragment.kt

```diff
@@ -145,4 +164,93 @@ class HolidayCalendarEditFragment() : UstadEditFragment<HolidayCalendar>(), Holi
     }
 
 
+}
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun HolidayCalendarEditScreen(
+    uiState: HolidayCalendarEditUiState,
+    onHolidayCalendarChange: (HolidayCalendar?) -> Unit = {},
+    onClickAddItem: () -> Unit = {},
+    onDeleteItemClick: (Holiday) -> Unit = {},
+    onItemClick: (Holiday?) -> Unit = {}
+){
+    Column (
+        modifier = Modifier
+            .padding(8.dp)
+            .verticalScroll(rememberScrollState()),
+        horizontalAlignment = Alignment.CenterHorizontally
+    ){
+
+        UstadTextEditField(
+            value = uiState.holidayCalendar?.umCalendarName ?: "",
+            label = stringResource(id = R.string.name),
+            enabled = uiState.fieldsEnabled,
+            onValueChange = {
+                onHolidayCalendarChange(uiState.holidayCalendar?.shallowCopy {
+                    umCalendarName = it
+                })
+            }
+        )
+
+        ListItem(
+            modifier = Modifier.clickable {
+                onClickAddItem()
+            },
+            icon = {
+                Icon(
+                    Icons.Filled.Add,
+                    contentDescription = null
+                )
+            },
+            text = { Text(stringResource(id = R.string.add_a_holiday)) }
+        )
+
+        uiState.holidayList?.forEach { holiday ->
+
+            val holidayStart = rememberFormattedDate(timeInMillis = holiday.holStartTime, timeZoneId = "UTC")
+            val holidayEnd = rememberFormattedDate(timeInMillis = holiday.holEndTime, timeZoneId = "UTC")
+
+            ListItem(
+                modifier = Modifier
```

Comment:
The holiday itself does not have an icon, but you need to use a blank icon so that the spacing will be correct and the text will line up. The text for "add a holiday" and "Eid" should align, 

![spacer](https://user-images.githubusercontent.com/4280622/209865796-28adcf99-f8cd-47ac-a80e-5cc03aa9c511.png)

The way to do this is to add a blank icon so that the text will start at the same position.

e.g. add this to the listitem.

```
ListItemIcon {
   UstadBlankIcon { } 
}
```